scripts/bench/bench_version.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+# Benchmark between 2 existing versions
+# Usage: ./scripts/bench/bench_version.sh gosec v1.58.1 v1.58.2
+
+# ex: gosec
+LINTER="$1"
+
+# ex: v1.58.1
+VERSION_OLD="$2"
+# ex: v1.58.2
+VERSION_NEW="$3"
```

Comment:
Maybe add some guards or usage after this if someone just runs the target or script instead of the top comment?

```sh
if [ -z "$LINTER" ] || [ -z "$VERSION_OLD" ] || [ -z "$VERSION_NEW" ]; then
  cat <<-EOF
Missing required arguments!

Usage:   $0 <linter> <old version> <new version>
Example: $0 gosec v1.58.1 v1.58.2
EOF

  exit 1
fi

EOF